packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding_phone_confirmation/blocs/onboarding_phone_bloc.dart

```diff
@@ -69,24 +67,30 @@ class OnboardingPhoneBloc extends $OnboardingPhoneBloc {
   /// The navigation bloc used to navigate the user
   final RouterBlocType _navigationBloc;
 
+  /// The initial country code to be presented. Most apps default to the
+  /// US country code.
+  final _initialCountryCode = CountryCodeModel(code: '1', name: 'USA');
```

Comment:
My only suggestion is to maybe move this to a `CountryCodeModel.withDefaults()` factory inside the model itself